include/exadg/functions_and_boundary_conditions/function_cached.h

```diff
@@ -71,16 +73,16 @@ class FunctionCached
   }
 
   void
-  set_data_pointer(std::map<unsigned int, MapVectorIndex> const & global_map_vector_index_in,
-                   std::map<unsigned int, ArrayTensor> const &    map_solution_in)
+  set_data_pointer(std::map<unsigned int, MapVectorIndex> const &      map_map_vector_index_,
+                   std::map<unsigned int, ArraySolutionValues> const & map_array_solution_)
   {
-    global_map_vector_index = &global_map_vector_index_in;
-    map_solution            = &map_solution_in;
+    map_map_vector_index = &map_map_vector_index_;
+    map_array_solution   = &map_array_solution_;
   }
 
 private:
-  std::map<unsigned int, MapVectorIndex> const * global_map_vector_index;
-  std::map<unsigned int, ArrayTensor> const *    map_solution;
+  std::map<unsigned int, MapVectorIndex> const *      map_map_vector_index;
+  std::map<unsigned int, ArraySolutionValues> const * map_array_solution;
```

Comment:
These variable names really don't help to understand the meaning. What about given a name that is more descriptive or alternatively add a comment!?

Reply:
I agree that it is not easy to find good names. However, I think that the names have at least improved compared to the previous version (For example, I was irritated by the name `global_map_...` in the previous version, which initiated the renaming performed here). I don't know if there are general C++ recommendations how to name maps ... if there are suggestion, we can of course further improve the names.

Reply:
The name only indicates the type of the variable. For that one has an IDE. If someone takes a look at the code, he/she would like to know what is mapped without the need to look into the setup routines.